src/span.rs

```diff
@@ -213,11 +217,12 @@ impl Default for SpanContextStateBuilder {
     }
 }
 
-/// Jager specific span context state.
+/// Jaeger specific span context state.
 #[derive(Debug, Clone)]
 pub struct SpanContextState {
     trace_id: TraceId,
     span_id: u64,
+    parent_span_id: u64,
```

Comment:
Currently, `parent_span_id` have no actual meaning (cf. https://github.com/jaegertracing/jaeger/pull/831), so I think it is sufficient to remove the handling of this field and always set the dummy value `0` (as `sile/rustracing_jaeger/master` branch).

Reply:
Fixed.